tests/integration/test_storage_kafka/test_batch_fast.py

```diff
@@ -3082,70 +3085,82 @@ def test_block_based_formats_1(kafka_cluster, create_query_generator):
             ["40", "400"],
         ]
 
-
-def test_system_kafka_consumers(kafka_cluster):
+@pytest.mark.parametrize(
+    "create_query_generator, consumer_id_length, num_rebalance_assignments",
+    [(k.generate_old_create_table_query, 67, 1), (k.generate_new_create_table_query, 0, 0)],
+)
+def test_system_kafka_consumers(kafka_cluster, create_query_generator, consumer_id_length, num_rebalance_assignments):
```

Comment:
@pamarcos The test is here. The trick is `generate_new_create_table_query` will generate a create query that will result in creating `StorageKafka2`.